examples/python/phi3v.py

```diff
@@ -83,5 +107,14 @@ def run(args: argparse.Namespace):
     parser.add_argument(
         "-p", "--provider", type=str, required=True, help="Provider to run model"
     )
+    parser.add_argument(
+        "--image_paths", nargs='*', type=str, required=False, help="Path to the images"
+    )
+    parser.add_argument(
+        '-pr', '--prompt', required=False, help='Input prompts to generate tokens from.'
+    )
+    parser.add_argument(
+        '--interactive', default=False, required=False, help='Interactive mode'
```

Comment:
Can we keep the existing behavior as the default? Users will be referring to this script when running the new vision models and our published examples on [the ORT website](https://onnxruntime.ai/docs/genai/tutorials/phi3-v.html) and on [GitHub](https://github.com/microsoft/onnxruntime-genai/blob/main/examples/python/phi-3.5-vision.md#4-run-phi-35-vision-onnx-models) refer to `-m` and `p` as the only required flags.